app/views/posts/show.html.erb

```diff
@@ -41,62 +43,74 @@
                   method: :delete,
                   data: { confirm: "Are you sure?" } %>
         </div>
-        <% if ((@post.account_id == current_account.id) and !(@post.closed?)) or !(@post.closed?) %>
+      <% end %>
+         <% if ((@post.account_id == current_account.id) and !(@post.closed?)) or !(@post.closed?) %>
           <div class="join-btn" id="close-link">
             <%= link_to "Close Discussion", close_post_path(@post), method: :patch %>
           </div>
-        <% end %>
+         <% end %>
+        <% end %> 
         <button type="button" class="join-btn" data-toggle="modal" data-target="#myModal" remote = "true">
           Report
        </button>
       </div>
 	</div>  
-<div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
-  <div class="modal-dialog" role="document">
-    <div class="modal-content">
+   <div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```

Comment:
Can you move the modal to a separate partial?

Reply:
Yes